birdhouse/env.local.example

```diff
@@ -208,8 +229,10 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 
 # Jupyter single-user server images
 #export DOCKER_NOTEBOOK_IMAGES="pavics/workflow-tests:210216 \
-#                               pavics/crim-jupyter-eo:0.1.0 \
-#                               pavics/crim-jupyter-nlp:0.1.0"
+#                               ${DOCKER_EO_IMAGE} \
+#                               ${DOCKER_NLP_IMAGE}"
+#export DOCKER_EO_IMAGE="pavics/crim-jupyter-eo:0.1.0"
+#export DOCKER_NLP_IMAGE="pavics/crim-jupyter-nlp:0.1.0"
```

Comment:
These vars `DOCKER_EO_IMAGE` and `DOCKER_NLP_IMAGE` have to be declared before `DOCKER_NOTEBOOK_IMAGES` that uses them.